src/league_scraper_v1.py

```diff
@@ -0,0 +1,60 @@
+import pprint
+import re #regular expression
+import time
+from selenium import webdriver
+from bs4 import BeautifulSoup
+
+def scrapeMatchHistoryUrl(usernametext, passwordtext, matchUrl):
+
+	#options = webdriver.ChromeOptions()
+	#options.add_argument('--headless')
+	browser = webdriver.Chrome()
+	browser.maximize_window()
+	browser.get(matchUrl)
+	signinbutton = browser.find_element_by_css_selector(".riotbar-account-action")
+	signinbutton.click()
+
+
+	username = browser.find_element_by_name("username")
+	username.clear();
+	username.send_keys(usernametext)
+
+	password = browser.find_element_by_name("password")
+	password.clear();
+	password.send_keys(passwordtext)
+
+	submit = browser.find_element_by_css_selector(".mobile-button__submit")
+	submit.click()
+
+	time.sleep(8) #sleep while page loads
+
+	page_source = browser.page_source
+	soup = BeautifulSoup(page_source, 'html.parser')
+
+	browser.close()
+
+	#test print statements
+	#print(soup.prettify())
```

Comment:
suggest removing comments or uncommenting

Reply:
leave them for now, will revisit and use these print statements when updating scraper to get more data from each match